tests/test_mock_sampler.py

```diff
@@ -14,19 +14,88 @@
 
 """who watches the watchmen?"""
 import unittest
+import os
+import dimod.testing as dit
 
+from dwave.system import DWaveSampler
 from dwave.system.testing import MockDWaveSampler, MockLeapHybridDQMSampler
-
-import dimod.testing as dit
+from dwave.cloud.exceptions import ConfigFileError, SolverNotFoundError
 from dimod import DiscreteQuadraticModel, ExtendedVartype, SampleSet
 
-
+@unittest.skipIf(os.getenv('SKIP_INT_TESTS'), "Skipping integration test.")
 class TestMockDWaveSampler(unittest.TestCase):
+    def test_properties_and_params(self):
```

Comment:
Let's move `skipIf` to the specific test below only, that way all the other tests will run even when access to SAPI is not available. (Ideally, we should be able to test the mock sampler offline, or test the majority of it.)